src/libData/AccountData/Account.cpp

```diff
@@ -164,11 +164,7 @@ bool Account::InitContract(const bytes& code, const bytes& initData,
     string vname = v["vname"].asString();
     string type = v["type"].asString();
 
-    Json::StreamWriterBuilder writeBuilder;
-    std::unique_ptr<Json::StreamWriter> writer(writeBuilder.newStreamWriter());
-    ostringstream oss;
-    writer->write(v["value"], &oss);
-    string value = oss.str();
+    string value = JSONUtils::convertJsontoStr(v["value"]);
```

Comment:
This just got my attention as the `Json::StreamWriterBuilder` and `Json::StreamWriter` are created and destroyed on the heap for every `InitContract` call. This remains true after replacing the entire block with `JSONUtils:: convertJsontoStr`. Are we sure this is an efficient way of using these two functions? There're some discussion [here](https://github.com/open-source-parsers/jsoncpp/issues/658#issuecomment-333073996) about the performance.